packages/prettier-config/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 -   Increase the minimum Node.js version to 14 ([#43141](https://github.com/WordPress/gutenberg/pull/43141)).
 
+### Bug Fix
+
+-   Packages: Replace `is-plain-obj` with `is-plain-object` ([#43511](https://github.com/WordPress/gutenberg/pull/43511)).
```

Comment:
Same.

Reply:
👍 Addressed as part of 3b02f0048724f064dcd1df8f6968d7e38c1565fb.